index.bs

```diff
@@ -324,13 +325,60 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: A user agent implementing, in conjunction with the underlying platform, the [=Web Authentication API=] and algorithms
     given in this specification, and handling communication between [=authenticators=] and [=[RPS]=].
 
+: <dfn>Credential ID</dfn>
+:: A probabilistically-unique [=byte sequence=] identifying a [=public key credential source=].
```

Comment:
the way we use credential in the spec today suggests that credential actually refers to the authentication key pair and not the authentication assertion.
authenticatorMakeCredential makes a credential i.e. the key pair.

Reply:
when ever we mean the assertion we use the term assertion (not credential)

Reply:
Credential ID is already defined in the spec at this point.  This addition can be deleted at this point.